### PR TITLE
Add Librispeech French corpus

### PR DESCRIPTION
**Pull request by hirofumi0810**
- [x] add speech translation recipe
- [x] add asr recipe
- [x] add results

### CODE REVIEW

**Comment by hirofumi0810** [comment]
Download data from here:
`https://persyval-platform.univ-grenoble-alpes.fr/DS91/detaildataset`
Note that this dataset cannot be downloaded automatically because e-mail based comfirmation is required before download.

**Comment by sw005320** [comment]
- `egs/librispeech_french/asr1/run_asr.sh` -> `egs/librispeech_french/asr1/run.sh`?

Also, maybe it's nonsense, but you can think of moving some common speech translation functions (e.g., `local/score_bleu.sh`) to `utils`? Then, you can add some test functions, etc., and make your scripts reusable and keep them clean.

**Comment by sw005320** [comment]
> Download data from here:
https://persyval-platform.univ-grenoble-alpes.fr/DS91/detaildataset
Note that this dataset cannot be downloaded automatically because e-mail based comfirmation is required before download.

Thanks, can you add such descriptions in the data directory part of `run.sh`?

**Comment by hirofumi0810** [comment]
> Also, maybe it's nonsense, but you can think of moving some common speech translation functions (e.g., `local/score_bleu.sh`) to `utils`? Then, you can add some test functions, etc., and make your scripts reusable and keep them clean.

It's difficult to create a unified score_bleu.sh because normalization is different slightly depending on corpora.
Let me do it in another PR.
I mean... we need careful discussion about it.

File: egs/librispeech_french/asr1/run.sh [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+
+# preprocessing related
+case=lc.rm
+# tc: truecase
+# lc: lowercase
+# lc.rm: lowercase with punctuation removal
+
+# Set this to somewhere where you want to put your data, or where
+# someone else has already put it.  You'll want to change this
+# if you're not on the CLSP grid.
```
Comment (sw005320) [inline egs/librispeech_french/asr1/run.sh]: This path is not CLSP one.

**Comment by hirofumi0810** [comment]
> LGTM.
> We're now downloading the data and will test it.
> I could merge this PR and can ask you to refine it later if there are some issues.
> @hirofumi0810, what do you think?

I can update speech translation recipes to keep consistency in other PRs.

File: egs/librispeech_french/asr1/run.sh (hunk 1 above)
Comment (hirofumi0810) [reply egs/librispeech_french/asr1/run.sh]: @sw005320  Could you tell me the absolute path where you downloaded data?

**Comment by sw005320** [comment]
OK. I decided to merge it. Once you fix the above comment part, I'll do it.

File: egs/librispeech_french/asr1/run.sh (hunk 1 above)
Comment (sw005320) [reply egs/librispeech_french/asr1/run.sh]: We don't finish the download yet, and I can do it later. So, you can just replace the above `CLSP grid` with your lab's grid (or remove this sentence).

**Pull request merged**